Misc/NEWS.d/next/Core and Builtins/2018-12-22-22-19-51.bpo-35560.9vMWSP.rst

```diff
@@ -0,0 +1,3 @@
+Removed an incorrect assert statement that caused segfault in :func:`format`
+in debug builds for floating point formatting with zero padding and small width.
```

Comment:
You should mention that the only the "n" format is impacted by the bug. I propose:

Fix an assertion error in :func:`format` in debug build for floating point formatting with "n" format, zero padding and small width. Release build is not impacted. Patch by Karthikeyan Singaravelan.

Reply:
Thanks Victor for the suggestion I have reworded the entry.